src/factory/factory.py

```diff
@@ -3,7 +3,7 @@
 """
 
 from ..agent import Planner, Search_agent, Reporter, RAG_agent
-
+from ..agent.notion import NotionAgent
```

Comment:
add NotionAgent to __init__.py so that we could 
```suggestion
from ..agent import Planner, Search_agent, Reporter, RAG_agent, NotionAgent
```

Reply:
there is no notion agent ? Maybe create a class first or whole project is not runnable